packages/application-components/src/components/modal-pages/internals/modal-page.styles.ts

```diff
@@ -47,7 +55,7 @@ export const getBeforeCloseOverlayAnimation = () => css`
   opacity: 0 !important;
 `;
 
-export const ContentWrapper = styled.div`
+export const ContentWrapper = styled('div')`
```

Comment:
is this because @emotion/styled is missing typescript defs?

Reply:
It does have type defs, this is how it’s documented in their docs: https://emotion.sh/docs/typescript#emotionstyled

Reply:
well in the link they also use `styled.div`

```js
// TS 2.9+ only
const Image3 = styled.div<ImageProps>`
  width: ${(props: ImageProps) => props.width};
  background: url(${(props: ImageProps) => props.src})
    center center;
  background-size: contain;
`
```

Reply:
I'll give it another try

Reply:
Thanks! https://github.com/commercetools/merchant-center-application-kit/pull/921/commits/0e1ad7f7406c1f2687d99ed022dde833339beb43